boards/posix/doc/arch_soc.rst

```diff
@@ -350,10 +350,10 @@ and this thread will check what is the next
 scheduled HW event, advance simulated time until that point, and call the
 corresponding HW model event function.
 
-Eventually one of these HW models will raise an interrupt to the simulated CPU.
-When the IRQ controller wants to wake the simulated CPU, the HW thread is
-blocked, and the simulated CPU is awaken by letting the last SW thread continue
-executing.
+Eventually one of these HW models will raise an interrupt to the
+simulated CPU.  When the IRQ controller wants to wake the simulated
```

Comment:
double space after dot

Reply:
Very sorry for the oversight when I reformatted that paragraph! Shall I rebase and push again as per the instructions in https://docs.zephyrproject.org/latest/contribute/guidelines.html ?

Reply:
Yes. You can force push to the branch an amended commit and the PR will be updated automatically.

Reply:
I am afraid I'm not used to the workflow and must have deleted the original commit. I have opened a new pull request (#54891) instead. Sorry for the confusion!